### PR TITLE
Fixed json formatting

### PR DESCRIPTION
**Pull request by myun11**
The API gives an error when you query sword-of-narzissenkreuz so I fixed the json title from "en copy 3.json" to "en.json".

Also I added its missing AscensionMaterial.

### CODE REVIEW

File: assets/data/weapons/the-catch/en,json
Comment (niktekusho) [inline assets/data/weapons/the-catch/en,json:1]: I don't think this file was intended to be renamed (`en.json` -> `en,json`)

Comment (man90es) [reply assets/data/weapons/the-catch/en,json:1]: The other way around, but good job spotting this

Comment (myun11) [reply assets/data/weapons/the-catch/en,json:1]: I fixed it to be `en.json`

**Review by man90es — APPROVED** [review]
Looks good to me, thank you for your contribution

**Pull request merged**